docs/dg/dev/third-party-module-development/ensure-compatibility.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Ensure compatibility
+description: Ensure compatibility
+last_updated: Jun 7, 2024
+template: howto-guide-template
+
+---
+
+## Code compatibility
+Steps how to ensure code compatibility with latest Spryker releases:
+
+**1.** Decide which demoshop is most relevant for your module testing, choose one of:
+- https://github.com/spryker-shop/b2c-demo-[shop](https://github.com/spryker-shop/b2c-demo-shop/blob/master/.github/workflows/ci.yml)
+- https://github.com/spryker-shop/b2b-demo-shop
+- https://github.com/spryker-shop/b2c-demo-marketplace
+- https://github.com/spryker-shop/b2b-demo-marketplace
+
+**2.** Add to you module CI pipeline that does the following thing:
```

Comment:
your module a CI

Reply:
It will mean the different thing than, i would leave it as it is. What do you think?